GW2EIEvtcParser/EIData/InstantCastFinders/MissileCastFinder/MissileCastFinder.cs

```diff
@@ -4,16 +4,29 @@ namespace GW2EIEvtcParser.EIData;
 
 internal class MissileCastFinder : CheckedCastFinder<MissileEvent>
 {
+    protected bool Minions = false;
     private readonly long _missileSkillID;
+    private int _speciesId = 0;
```

Comment:
This does not look like it is serving any purpose?

Reply:
Removed it